documentation/modules/ref-operator-cluster.adoc

```diff
@@ -121,6 +118,31 @@ The DNS domain name is added to the Kafka broker certificates used for hostname
 +
 If you are using a different DNS domain name suffix in your cluster, change the `KUBERNETES_SERVICE_DNS_DOMAIN` environment variable from the default to the one you are using in order to establish a connection with the Kafka brokers.
 
+== Logging configuration
+
+When installing Cluster Operator, `ConfigMap` containing its logging configuration is created.
+This `ConfigMap` is described in the file `install/cluster-operator/050-ConfigMap-strimzi-cluster-operator-logging.yaml`.
+Cluster Operator logging is then configured by changing this `ConfigMap`.
+
+You can change the file and then use:
+[source,shell,subs=+quotes]
+kubectl apply -f _install/cluster-operator/050-ConfigMap-strimzi-cluster-operator-logging.yaml_
+
+Or you can edit the `ConfigMap` by:
+[source,shell,subs=+quotes]
+kubectl edit cm strimzi-cluster-operator-logging
+
+After the changes are done in the `ConfigMap`, the logging configuration is reloaded in time interval of specified `monitorInterval` seconds.
+
+NOTE: Created `ConfigMap` contains entry `monitorInterval`.
+Please ensure yourself you are not removing this entry as it serves for dynamic reloading of logging configuration.
```

Comment:
```suggestion
Check you are not removing this entry as it serves for dynamic reloading of logging configuration.
```

Reply:
Removed in https://github.com/strimzi/strimzi-kafka-operator/pull/3328#discussion_r460786285